ComposeAdvanced/app/src/main/java/com/example/android/wearable/composeadvanced/presentation/ui/userinput/UserInputComponentsScreen.kt

```diff
@@ -143,5 +154,45 @@ fun UserInputComponentsScreen(
                 }
             )
         }
+
+        item {
+            val launcher =
+                rememberLauncherForActivityResult(
+                    ActivityResultContracts.StartActivityForResult()
+                ) {
+                    it.data?.let { data ->
+                        val results: Bundle = RemoteInput.getResultsFromIntent(data)
+                        val newInputText: CharSequence? = results.getCharSequence("input_text")
+                        textForUserInput = newInputText as String
+                    }
+                }
+            val label = stringResource(R.string.manual_text_entry_label)
+            Chip(
+                onClick = {
+                    val intent: Intent = RemoteInputIntentHelper.createActionRemoteInputIntent()
+                    val remoteInputs: List<RemoteInput> = listOf(
+                        RemoteInput.Builder("input_text")
+                            .setLabel(label)
```

Comment:
Maybe show 

```
                .wearableExtender {
                      setEmojisAllowed(true)
                    setInputActionType(EditorInfo.IME_ACTION_DONE)
                }
```

Reply:
+1, was also going to suggest it

Reply:
As discussed added a dependency on the Wear-input 1.2.0-alpha02 library and added wearableExtender